lib/widgets/inbox.dart

```diff
@@ -368,18 +369,21 @@ class _DmItem extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final store = PerAccountStoreWidget.of(context);
+    final zulipLocalizations = ZulipLocalizations.of(context);
     final selfUser = store.users[store.selfUserId]!;
 
     final designVariables = DesignVariables.of(context);
 
     final title = switch (narrow.otherRecipientIds) { // TODO dedupe with [RecentDmConversationsItem]
       [] => selfUser.fullName,
-      [var otherUserId] => store.users[otherUserId]?.fullName ?? '(unknown user)',
+      [var otherUserId] => store.users[otherUserId]?.fullName ?? zulipLocalizations.unknownUserName,
 
       // TODO(i18n): List formatting, like you can do in JavaScript:
       //   new Intl.ListFormat('ja').format(['Chris', 'Greg', 'Alya', 'Shu'])
       //   // 'Chris、Greg、Alya、Shu'
-      _ => narrow.otherRecipientIds.map((id) => store.users[id]?.fullName ?? '(unknown user)').join(', '),
+      _ => narrow.otherRecipientIds.map((id) =>
+            store.users[id]?.fullName ?? zulipLocalizations.unknownUserName
```

Comment:
bump: we use 2 spaces for indentation

Reply:
![image](https://github.com/user-attachments/assets/fe4342b4-27b4-4ce3-a3f0-ddc7d0aca29a)

I am using android studio as my IDE and i configured it to use 2 spaces, and the changes I provided used this 2 spaces for indentation

Reply:
I see. That should work for most cases automatically, but looks like at the line (started with `store.users[id]?.fullName`) I commented, there is only one space indented from the text it aligns to (started with `narrow.otherRecipientIds.map`) in the previous line.

You can count the number of spaces at the column where the `n` from `narrow.otherRecipientIds.map` is at, at line `store.users[id]?.fullName` to see what I mean.


I would suggest the following change:

before:
```dart
      _ => narrow.otherRecipientIds.map((id) =>
            store.users[id]?.fullName ?? zulipLocalizations.unknownUserName
```

after:
```dart
      _ => narrow.otherRecipientIds.map((id) =>
             store.users[id]?.fullName ?? zulipLocalizations.unknownUserName
```

Reply:
I was able to add the changes as requested and i updated the PR description.